release.nix

```diff
@@ -58,30 +61,36 @@ let
       (builtins.attrValues ghcjs)
       (concatDepends ghc)
       (concatDepends ghcjs)
-      [command serverExeSkeleton iosSkeleton androidSkeleton]
+      (lib.optional reflex-platform.androidSupport androidSkeleton)
+      (lib.optional reflex-platform.iosSupport iosSkeleton)
+      [ command serverSkeletonExe serverSkeletonShell ]
     ];
-    command = local-self.command;
-    serverExeSkeleton = (import ./skeleton { obelisk = local-self; }).exe;
-    androidObelisk = import ./. (self-args // {
-      config.android_sdk.accept_license = true;
-    });
-    androidSkeleton = (import ./skeleton { obelisk = androidObelisk; }).android.frontend;
-    iosObelisk = import ./. (self-args // {
-      system = "x86_64-darwin";
-      iosSdkVersion = "10.2";
-    });
-    iosSkeleton = (import ./skeleton { obelisk = iosObelisk; }).ios.frontend;
+    command = obelisk.command;
+    skeleton = import ./skeleton { inherit obelisk; };
+    serverSkeletonExe = skeleton.exe;
+    # TODO fix nixpkgs so it doesn't try to run the result of haskell shells as setup hooks.
+    serverSkeletonShell = local-self.nixpkgs.runCommand "shell-safe-for-dep" {} ''
```

Comment:
stdenv.sh in nixpkgs will do a bash `source` of any depedendeny which is a file rather than directory (ugh!). `nix-shell` builds a single file, which isn't bash, and that blows up the build.